test/e2e-go/features/catchup/catchpointCatchup_test.go

```diff
@@ -135,11 +137,13 @@ func TestBasicCatchpointCatchup(t *testing.T) {
 	cfg, err := config.LoadConfigFromDisk(primaryNode.GetDataDir())
 	a.NoError(err)
 	cfg.CatchpointInterval = 4
+	cfg.MaxAcctLookback = 2
 	cfg.SaveToDisk(primaryNode.GetDataDir())
 	cfg.Archival = false
 	cfg.NetAddress = ""
 	cfg.EnableLedgerService = false
 	cfg.EnableBlockService = false
+	cfg.MaxAcctLookback = 2
```

Comment:
this is the second node, and if has the default `MaxAcctLookback=8` if might fail to get some blocks from relays because of other consensus and config values

Reply:
But the data structure should have the value already set since we are using the same cfg variable.

Reply:
ah, right, I'll update